docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -0,0 +1,215 @@
+
+To prevent these types of exfiltration attacks altogether, Teleport clients support hardware-based private keys. Unlike disk-based private keys, hardware-based private keys are generated and stored directly on a hardware device and are impossible to export. With this features, a user's login session is only functional with access to the user's hardware device.
+
+Additionally, this feature can be configured to require touch for every Teleport request, including non-session requests. With touch required, Hardware Key Support provides better security in comparison to our per-session MFA offering.
+
+<Admonition type="note" title="touch caching">
+
+  A user's touch is cached on the YubiKey for 15 seconds to prevent excessive touch prompts.
+
+</Admonition>
+
+<ScopedBlock scope="oss">
+
+  Hardware-backed Login Sessions require Teleport Enterprise or Teleport Cloud.
+
+</ScopedBlock>
+
+<Admonition type="note" title="PIV support">
+
+  Hardware-backed Login Sessions require a PIV-compatible hardware key. Currently, this feature is only guarenteed to support YubiKey series 5+.
+
+</Admonition>
+
+<Admonition type="warning" title="Compatibility Issues">
+
+  Hardware-backed Login Sessions provides the best security available, but it is a new feature with some compatiblity trade-offs. We hope to remove these incompatibilities as development progresses.
+  
+  Currently, Hardware-backed Login Sessions support `tsh`, `tctl`, and Teleport Connect for:
+    
+    - Standard Teleport API requests (`tsh ls`, `tctl create`, etc.)
+    - Server Access
+    - Database Access (using `tsh proxy db` instead of `tsh db connect`)
+    - Application Access
+    - Desktop Access
+  
+  Hardware-backed Login sessions do not yet support:
+
+    - Teleport WebUI (except for user registration / reset password)
+    - Agent forwarding functionality such as `tsh ssh -A`, Proxy Recording mode, and OpenSSH integration
+    - Kubernetes Access
+
+  To navigate these incompatiblities, we recommend enabled Hardware Key support only when necessary, such as for roles with access to critical infrastructure. These roles can be accessed as needed with Access Requests so that users can avoid these issues for their normal login sessions.
+
+</Admonition>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A series 5+ YubiKey
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Enable Hardware Key support
+
+Hardware Key support is disbled by default. You can enable Hardware Key support for specific roles like below:
+
+```yaml
+kind: role
+metadata:
+  name: admin
+spec:
+  allow:
+    require_session_mfa: hardware_key_touch
```

Comment:
Please also update the role reference page to list the valid values for `require_session_mfa`. I'm pretty sure it used to be a boolean, if that's changed we should highlight it and document all possible values.

Reply:
I don't see a place to list the valid values, but I updated the references's description and value type.